unittests/resources/checks_unlisted/cleanup_fail.py

```diff
@@ -0,0 +1,28 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class CleanupFailTest(rfm.RunOnlyRegressionTest):
```

Comment:
Consider moving this inside the `unittests/resources/checks/frontend_checks.py`. You will not then need the `SleepTest`, which is duplicate, you won't need any other unit test, since the standards execution policy unit tests should cover you. The only thing you need is to adjust the expected failures in the corresponding unit tests.